copter/source/docs/parachute.rst

```diff
@@ -37,39 +37,57 @@ Connecting to the Autopilot
 ===========================
 
 
-The parachute release mechanism can be triggered from either a Relay or a PWM (i.e. Servo) but because of `an issue with the Relay pins being pulled high at start-up <https://github.com/ArduPilot/ardupilot/issues/1239>`__, 
-we recommend using PWM, in particular any of the Pixhawk's AUX OUT 1 to 4 pins.
+The parachute release mechanism can be triggered from either a Relay or a PWM (i.e. Servo). Be sure that if a Relay pin is used, that the parachute release is active on a high level and that the Relay pin default, :ref:`RELAY_DEFAULT<RELAY_DEFAULT>` is OFF or No Change, to avoid triggering the release during initialization, since all GPIOs are forced low during the bootloader period.
+
 
 .. image:: ../images/Parachute_Pixhawk.jpg
     :target: ../_images/Parachute_Pixhawk.jpg
 
-Setup through the mission planner
+Setup through the Mission Planner
 =================================
 
-To configure the parachute release please first connect with the Mission
-Planner and then open the Config/Tuning >> Full Parmeter List page and
+To configure the parachute release, first connect to the autopilot with Mission
+Planner, and then open the CONFIG->Full Parameter List tab and
 set the following parameters.
 
--  :ref:`CHUTE_ENABLED <CHUTE_ENABLED>` = "1"
+-  :ref:`CHUTE_ENABLED <CHUTE_ENABLED>` = "1". Then refresh parameters, or reboot, to see the Chute parameters.
+-  :ref:`CHUTE_CRT_SINK<CHUTE_CRT_SINK>` should be set to the sink rate in m/s that will trigger the chute release.
```

Comment:
although the parameter description doesn't mention this, it seems to be that if left at zero the sink rate check is disabled (meaning any sink rate is ok).